### ISSUE TITLE
[Forge 1.20.1] Crash Upon Generating Hanging Signs

### ISSUE THREAD

**Issue by FoxMcloud5655** [MehVahdJukaar/Supplementaries#723]
MC version: 1.20.1
Forge version: 47.1.3
Mod version: 2.5.21

One of my players is crashing my dedicated server when they venture out and generate a hanging sign from Supplementaries.

Relevent part of the crash log:
```
Description: Feature placement

java.lang.ClassCastException: class net.minecraft.server.level.WorldGenRegion cannot be cast to class net.minecraft.world.level.Level (net.minecraft.server.level.WorldGenRegion and net.minecraft.world.level.Level are in module minecraft@1.20.1 of loader 'TRANSFORMER' @1d2c253)
	at net.mehvahdjukaar.supplementaries.common.block.tiles.HangingSignTileExtension.updateShape(HangingSignTileExtension.java:110) ~[supplementaries-1.20-2.5.21.jar%23550!/:?] {re:mixin,re:classloading}
	at net.minecraft.world.level.block.WallHangingSignBlock.handler$zdc002$updateShape(WallHangingSignBlock.java:539) ~[server-1.20.1-20230612.114412-srg.jar%23573!/:?] {re:mixin,re:classloading,pl:mixin:APP:supplementaries-common.mixins.json:WallHangingSignBlockMixin,pl:mixin:A}
	at net.minecraft.world.level.block.WallHangingSignBlock.m_7417_(WallHangingSignBlock.java) ~[server-1.20.1-20230612.114412-srg.jar%23573!/:?] {re:mixin,re:classloading,pl:mixin:APP:supplementaries-common.mixins.json:WallHangingSignBlockMixin,pl:mixin:A}
```
The rest of the crash log can be attached if needed.

**Comment by FoxMcloud5655** [comment]
Fixed in 7dc84e8.

**Comment by MehVahdJukaar** [comment]
Wait how did you know xD. Can't update yet tho cause vacation, trying to delegate this

**Comment by FoxMcloud5655** [comment]
I'm somewhat of a Minecraft modder myself :)

**Comment by netrunnernumber37** [comment]
is there a way to fix worlds that were generated on 2.5.21?

**Comment by MehVahdJukaar** [comment]
> is there a way to fix worlds that were generated on 2.5.21?

What does that mean? Why are you commenting this on this closed issue? If you have a bug open a new issue with a latest.log
